robot/description/urdf/caffeine.gazebo.xacro

```diff
@@ -93,5 +93,56 @@
             <!-- TODO: add Gaussian noise based off Garmin GPS -->
         </plugin>
     </gazebo>
+    <!-- TODO: double check these numbers -->
+    <!-- Gazebo multicamera plugin -->
+    <gazebo reference="${tf_prefix}/camera_link">
+        <sensor type="multicamera" name="stereo_camera">
+            <update_rate>30.0</update_rate>
+            <camera name="left">
+            <horizontal_fov>1.571</horizontal_fov>
```

Comment:
Can you define any angles you use as constants in degrees (not radians) and when you use them, just apply (append) `*pi/180`